chunjun-examples/sql/postgresql/row_array_map_data_type.sql

```diff
@@ -0,0 +1,29 @@
+
+CREATE TABLE test
+(
+    id           bigint,
+    deployment_pictures ARRAY<varchar>,
```

Comment:
I tried to run the script, but a warning came up. 
```shell
2022-10-31 15:20:03,049 - 8088 WARN  [main] com.dtstack.chunjun.connector.jdbc.converter.JdbcRowConverter:Unsupported type: Map<String, String>
2022-10-31 15:20:03,056 - 8095 WARN  [main] com.dtstack.chunjun.connector.jdbc.converter.JdbcRowConverter:Unsupported type: Row(Name: String, Contact: String)
```

Do you know why

Reply:
Exclude nested data types, such as ROW(id int,data ROW(id string))

Reply:
because TypeConversions dose not support nested data type convert
![image](https://user-images.githubusercontent.com/43560964/200981428-ae029c31-63f8-4bd5-94dd-d2a81f33c522.png)

Reply:
@mggger